python/py-flatbuffers/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-flatbuffers
+# version should be the same as devel/flatbuffers
```

Comment:
Also add a comment to the flatbuffers port so that whichever one a developer looks at, they'll know they have to update the other one too.